package/MDAnalysis/exceptions.py

```diff
@@ -35,6 +35,26 @@ class SelectionError(Exception):
 class NoDataError(ValueError, AttributeError):
     """Raised when empty input is not allowed or required data are missing.
 
+    This exception is used in various scenarios:
+    -   Raised when a Topology Attribute (e.g., bonds, charges) is not
+        present in the data.
+
+    -   Raised when data is missing in an analysis class because the `run()`
+        method has not been called. Examples include polymer analysis, PSA
+        (Potential Surface Analysis), dielectric analysis, and hydrogen bond
+        analysis.
```

Comment:
These analyses refer to specific classes, could you please link to the classes in question? My original comment would have referred to the [module files containing them](https://github.com/MDAnalysis/mdanalysis/tree/develop/package/MDAnalysis/analysis). `psa` also is not potential surface analysis but path similarity analysis -- did you get "potential surface analysis" from our documentation somewhere? If so, please raise an issue so we can fix it!

Reply:
It was my mistake, sorry for that. It is Path Similarity Analysis.